### ISSUE TITLE
courses/library: support clearing of rating

### ISSUE THREAD

**Issue by Mutugiii** [open-learning-exchange/planet#7727]
Should we allow clearing/undoing of ratings?

**Comment by sahilvunnam** [comment]
Yes, but don't we already have this. Like a user can change a rating of a course even after they rated it. Maybe clearing the rating is a good idea, but undoing it is unnecessary since the user can change it.

**Comment by jessewashburn** [comment]
Clearing ratings would be helpful, in case someone rated a course by accident.

**Comment by Gavinp14** [comment]
There is currently no feature that allows for you to clear a rating on a course, but rather only change it. Would we want to implement this as its own button or just a simple click on the stars to clear it?

**Comment by Mutugiii** [comment]
If we were to use a reset button, it would be cleaner to have it in the ratings dialog to avoid making the courses table more cluttered, and potentially using an icon that communicates reset would be ideal

![image](https://github.com/user-attachments/assets/7007f1ab-1fb4-4fe1-ad7f-eb573a3a7753)

**Comment by Gavinp14** [comment]
I like that idea, what icon do you think would best suit this purpose?
